src/chrome/content/rules/Steam.xml

```diff
@@ -1,53 +1,56 @@
 <ruleset name="Steam">
-	<!-- "steam-api.com" has a cert but it doesn't resolve -->
-	<!--target host="steam-api.com" /-->
-	<target host="*.steam-api.com" />
-		<test url="http://community.steam-api.com/" />
-		<test url="http://community.steam-api.com/ISteamWebAPIUtil/GetSupportedAPIList/v1/" />
-		<test url="http://partner.steam-api.com/" />
+	<target host="community.steam-api.com" />
+	<target host="partner.steam-api.com" />
 
 	<target host="steamcommunity.com" />
-		<test url="http://steamcommunity.com/" />
-		<test url="http://steamcommunity.com/broadcast/getbroadcastmpd/?steamid=76561197966726415" />
-	<target host="*.steamcommunity.com" />
-		<test url="http://www.steamcommunity.com/" />
-		<test url="http://api.steamcommunity.com/" />
-		<test url="http://cdn.steamcommunity.com/public/css/skin_1/header.css" />
+	<target host="www.steamcommunity.com" />
+	<target host="api.steamcommunity.com" />
+	<target host="cdn.steamcommunity.com" />
 
+	<target host="steamgames.com" />
+	<target host="www.steamgames.com" />
 	<target host="partner.steamgames.com" />
 
+	<target host="steampowered.com" />
```

Comment:
Could you sort the subdomains according to our [contributing guide](https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#target-ordering)?

Reply:
Yes, done.